cmd/zoekt-sourcegraph-indexserver/grpc/protos/zoekt/indexserver/v1/indexserver.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
nit: we could flatten the directory structure a bit. I tried to follow what we already have, but it seems unnecessary 

We might as well choose something like this:
```
cmd/zoekt-sourcegraph-indexserver/grpc/indexserver/v1
cmd/zoekt-sourcegraph-indexserver/grpc/configuration/v1
```

Reply:
+1